### PR TITLE
Corresponds to a problem that abnormally ends at the time of crack ex…

### PR DESCRIPTION
**Pull request by Kerubito0**
…pansion when data exists outside the graph display.

### CODE REVIEW

**Comment by liuxuan30** [comment]
before merge, why there is NaN?

File: Source/Charts/Renderers/AxisRendererBase.swift [hunk 1]
```diff
@@ -90,6 +90,7 @@ open class AxisRendererBase: Renderer
     @objc open func computeAxisValues(min: Double, max: Double)
     {
         guard let axis = self.axis else { return }
+        if min.isNaN || max.isNaN { return }
```
Comment (jjatie) [inline Source/Charts/Renderers/AxisRendererBase.swift]: Why are you not adding to the list of guarded conditions?

**Comment by jjatie** [comment]
Is the NaN coming from your dataSet or an issue internal to the library?

**Comment by liuxuan30** [comment]
usually we guarantee that no NaN will come into computeAxisValues. so we have to know where is the NaN generated.

**Comment by jjatie** [comment]
Abandoned PR. Can be reopened when questions are answered.

**Pull request closed without merging**